server/package.json

```diff
@@ -8,7 +8,8 @@
     "start": "node src/index.js",
     "dev": "nodemon src/index.js",
     "lint:eslint": "eslint ./src --ext .js,.jsx,.ts,.tsx",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "prepare": "husky install"
```

Comment:
Second part of "hack" because of https://typicode.github.io/husky/#/?id=custom-directory,
in "prepare" you have to do like this: `"cd .. && husky install server/.husky"` to inform it to go one directory up (there is `.git `file) and then install it in server folder.

Then add here script for pretty-quick